src/index.css

```
@@ -2,10 +2,58 @@
 @theme {
-  --color-paper: #fffaf1; --color-ink: #173b2d; --color-eucalypt: #2f6a50; --color-blossom: #d95680; --color-peach: #f6d8bf; --color-harbour: #c4e1e7;
-  --font-display: Georgia, "Times New Roman", serif; --font-sans: "Avenir Next", Avenir, ui-sans-serif, system-ui, sans-serif;
+  --color-paper: #fffaf1;
+  --color-ink: #173b2d;
+  --color-eucalypt: #2f6a50;
+  --color-blossom: #d95680;
+  --color-berry: #a92f59;
+  --color-peach: #f6d8bf;
+  --color-pollen: #efbd68;
+  --color-harbour: #c4e1e7;
+  --font-display: Georgia, "Times New Roman", serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)